woocommerce/woocommerce-android
# Fix onboarding loading screen in dark mode

Parent issue #4413


### Description

Updates in-person-payments onboarding loading illustration in dark mode.

### Testing instructions

- Use a site that is eligible for in person payments
1. Switch your device to dark mode
2. Tap on App settings
3. Tap on in-person payments
4. Notice the illustration on the loading screen is updated

### Images/gif


| Before | After |
| -- | -- |
| ![Screenshot_20210809-120302_WooCommerce (Dev)](https://user-images.githubusercontent.com/2261188/128690289-6b44ebdd-dc1e-4eeb-a3f6-8c35c77cebb8.jpg) | ![Screenshot_20210809-120138_WooCommerce (Dev)](https://user-images.githubusercontent.com/2261188/128690300-a1597092-75fe-44cd-a16c-6ee1f93c36e2.jpg) |
